conf/deployment/azure/ipi_1az_rhcos_3m_3w.yaml

```diff
@@ -8,6 +8,10 @@ ENV_DATA:
   region: 'eastus'
   azure_base_domain_resource_group_name: 'ocsqe'
   base_domain: 'azure.qe.rh-ocs.com'
+  worker_availability_zones:
+    - '1'
```

Comment:
Isn't this zone name ?

Reply:
In Azure, the supported zones for location `eastus` are `1,2,3`